assert_nothing_raised(<+Exception+>) { <++> }
